Jenkinsfile

```diff
@@ -481,6 +481,17 @@ pipeline {
         }
       }
     }
+    stage('Lint'){
```

Comment:
I'm not sure if I'm reading the Jenkinsfile properly, but I guess this runs Lint after everything else - at least that's what was proposed in https://github.com/elastic/apm-server/issues/2342. What's the rationale for placing it late in the pipeline?

Reply:
TBH, I had this same question. I picked this issue up out of the backlog and so I went with the assumption that this point had been previously discussed. If this was not previously discussed, I agree that it should be moved earlier.

Reply:
Hmm, but now I'm wondering why are we moving it out of "Intake" at all :confused: 
@kuisathaverat do you recall the reasoning for splitting this out in https://github.com/elastic/apm-server/issues/2342?

Reply:
to have fast feedback on lint errors.

Reply:
OK, thanks. I'd suggest we make it the first stage then. I'm not super convinced this is worthwhile though, since Intake takes ~4m30s.

Reply:
I just picked this up out of our backlog, to be honest, so I'm not too committed to any course of action here.

It sounds to me like there isn't much appetite for this change and I agree that it isn't buying much here. I'm going to close this PR and the associated issue and if this concern arises again, we can start a new discussion.

Reply:
SGTM, thanks @cachedout!